### PR TITLE
Tweaking Styles/Classes

### PR DESCRIPTION
**Pull request by lmiller1990**
Need some extra classes for styling in Cohesive UI.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "cohesive-ui",
-  "version": "2.0.0",
+  "version": "2.0.1",
```
Comment (lmiller1990) [inline package.json:3]: Did a release to fix a tiny bug

File: src/SmartList/components/Header/Header.tsx [hunk 2]
```diff
@@ -90,11 +91,10 @@ const Header: React.FC<IProps> = ({ cols, selectAllCol, onSelectAll }) => {
     return !!x.handleSort && !(x.displayName && x.displayName === '')
   }
 
+  const klass = classnames(className, 'co-smart-list-header', 'pt-2 pb-2 d-flex align-items-center')
```
Comment (MichaelDorkhom) [inline src/SmartList/components/Header/Header.tsx:94]: `klass`?

Comment (lmiller1990) [reply src/SmartList/components/Header/Header.tsx:94]: `class` and `className` are already used!

Comment (MichaelDorkhom) [reply src/SmartList/components/Header/Header.tsx:94]: You don't need a variable for this

```
Row className={classnames(className, 'co-smart-list-header', 'pt-2 pb-2 d-flex align-items-center')}
```

Comment (julianjuko) [reply src/SmartList/components/Header/Header.tsx:94]: i love it

Comment (julianjuko) [reply src/SmartList/components/Header/Header.tsx:94]: (couldn't find any issues @lmiller1990 so if you address this then all good)

**Pull request merged**